microcloud/mdns/mdns.go

```diff
@@ -48,3 +50,18 @@ func dnsTXTSlice(list []byte) []string {
 
 	return parts
 }
+
+// CheckIPv6Status returns true if IPv6 is enabled, false otherwise.
+func CheckIPv6Status() (bool, error) {
+	allIPv6IfacesDisabled, err := ioutil.ReadFile("/proc/sys/net/ipv6/conf/all/disable_ipv6")
```

Comment:
If I recall correctly, the `ioutil` package is deprecated. The linter should have caught this. I'll add a ticket to investigate but in the meantime can you use `os.ReadFile` thanks.

Reply:
fixed